Agilefreaks/weatherios
# Feature/display user location

## Description

Use case
As a user I want to see my current location so that I know where the weather info is diplayed from

Acceptance criteria
Will diplay the user location in human readable form

## Related

https://trello.com/c/LIKoqywM

## Documentation

https://stackoverflow.com/questions/30401439/how-could-i-create-a-function-with-a-completion-handler-in-swift
https://blog.bobthedeveloper.io/completion-handlers-in-swift-with-bob-6a2a1a854dc4

## Link to demo

![Simulator Screen Shot - iPhone 12 - 2020-12-08 at 12 09 36](https://user-images.githubusercontent.com/15888842/101470110-3f781a00-394e-11eb-8d00-5fb1b472d961.png)

## Steps to Test or Reproduce

1. Run app on device
2. Accept location permissions
3. You should see details for current location (ex: City name, Coordinates: Lat, Long)
